site.yml

```diff
@@ -78,6 +78,22 @@
     - role: percona-backup
       tags: ['infra', 'percona']
 
+- name: install mongodb on primary
```

Comment:
This play name should change to reflect that it's for all hosts in the mongo_db group.